### ISSUE TITLE
Accepting refactor "type: ignore" puts newline in wrong spot

### ISSUE THREAD

**Issue by wardy3** [microsoft/pylance-release#3858]
Type: <b>Bug</b>

# Behaviour

This alerts that "exceptions" is not a known member and VSCode refactor allows you to ignore it.

```python
urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

However, instead of putting # type: ignore on the end of this line, it puts it at the beginning of the line below and can corrupt the code. At the very least, it doesn't suppress the error

```python
urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 # type: ignore
```

## Expected vs. Actual

I would expect the `# type: ignore` to be put on the end of the offending line

See also https://github.com/microsoft/vscode/issues/171955

## Steps to reproduce:

1. add a line such as the urllib3 one above
2. wait for the red wiggly line and the light bulb
3. select quick fix of adding type ignore

# Diagnostic data

-   Python version (& distribution if applicable, e.g. Anaconda): 3.9.0
-   Type of virtual environment used (e.g. conda, venv, virtualenv, etc.): Poetry
-   Value of the `python.languageServer` setting: Default

<details>

<summary>Output for <code>Python</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Python</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

<details>

<summary>User Settings</summary>

<p>

```

languageServer: "Pylance"

linting
• flake8Args: "<placeholder>"
• flake8Enabled: true
• mypyArgs: "<placeholder>"
• mypyEnabled: true

testing
• pytestEnabled: true

```

</p>
</details>

Extension version: 2023.1.10232135
VS Code version: Code - Insiders 1.75.0-insider (Universal) (f1b09237cd400d92987cf7a9681995e8011ef54c, 2023-01-23T11:52:31.652Z)
OS version: Darwin x64 22.2.0
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz (16 x 2300)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|6, 12, 16|
|Memory (System)|32.00GB (0.01GB free)|
|Process Argv|--crash-reporter-id e45c21e3-3270-4be4-bf1b-d02853448a5e|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256198
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30422396
pythontb:30258533
pythonptprofiler:30281269
vsdfh931:30280409
vshan820:30294714
pythondataviewer:30285072
vscod805cf:30301675
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30404738
cppdebug:30492333
vscaat:30438846
vsclangdf:30492506
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30618038
pylantcb52:30590116
pyindex848:30611229
nodejswelcome1:30587009
pyind779:30611226
pythonsymbol12:30622696
a9j8j154:30646983

```

</details>

**Comment by judej** [comment]
Thanks for the report. Could you please add the Pylance version info? I am unable to reproduce the problem while on Pylance v2023.1.30 or v2023.1.31 (Pre-Release)





![Image](https://user-images.githubusercontent.com/547753/214362253-475f80c4-5b56-4bf6-8832-fb7176d9bf22.gif)

**Comment by wardy3** [comment]
Hi. I am using the same pre-release but I just realised you need an extra line to trigger the bug

eg,
```python
import urllib3

urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
import logging
```

goes to

```python
import urllib3

urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 # type: ignoreimport logging
```

**Comment by wardy3** [comment]
```
Name: Pylance
Id: ms-python.vscode-pylance
Description: A performant, feature-rich language server for Python in VS Code
Version: 2023.1.41
Publisher: Microsoft
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance
```

**Comment by judej** [comment]
dupe of #3959

**Comment by bschnurr** [comment]
This issue has been fixed in prerelease version 2023.3.11, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023311-1-March-2023-PreRelease)
